tests/test_packageAlerts.py

```diff
@@ -288,22 +299,56 @@ def testCreateBBox(self):
         self.assertTrue(bbox == geom.Extent2I(self.cutoutSize,
                                               self.cutoutSize))
 
-    def testMakeCutoutBytes(self):
-        """Test round tripping an exposure/cutout to bytes and back.
+    def testCreateCcdDataCutout(self):
+        """Test that the data is being extracted into the CCDData cutout
+        correctly.
+        """
+        packageAlerts = PackageAlertsTask()
+
+        ccdData = packageAlerts.createCcdDataCutout(
+            self.exposure,
+            self.exposure.getWcs().getSkyOrigin(),
+            self.exposure.getPhotoCalib())
+        calibExposure = self.exposure.getPhotoCalib().calibrateImage(
+            self.exposure.getMaskedImage())
+
+        self.assertTrue(np.allclose(ccdData.wcs.wcs.cd,
```

Comment:
I think `self.assertFloatsAlmostEqual(ccdData.wcs.wcs.cd, self.cutoutWcs.wcs.cd)` might give more informative error messages if the test fails. The downside is you would have to have `TestPackageAlerts` inherit from `lsst.utils.tests.TestCase` instead of `unittest.TestCase`.

If you end up switching the assert here, also change the other `self.assertTrue(np.allclose(...` throughout this file.